src/python/WMCore/Services/DDM/__init__.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+"""
+_DDM.DDM_
+
+API for calling DDM service
+
+"""
+__all__ = ['DDM', 'DDMReqTemplate']
```

Comment:
The way I see it it's actually not really needed because both classes are "public". This might be the reason for the poor quality jenkins report...